### ISSUE TITLE
Export BPMN diagrams with nested namespaces

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#130]
The following diagram can be imported but will not be correctly exported:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:foo="http://foo" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn">
  <bpmn:process id="Process" isExecutable="false">
    <bpmn:task id="Task">
      <bpmn:extensionElements>
        <foo:config>
          <Entry key="A" value="B" />
          <Nested>
            <Entry key="A" value="B" />
          </Nested>
        </foo:config>
      </bpmn:extensionElements>
    </bpmn:task>
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process">
      <bpmndi:BPMNShape id="Task_di" bpmnElement="Task">
        <dc:Bounds x="276" y="80" width="100" height="80" />
      </bpmndi:BPMNShape>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
```

The problem is that the BPMN export does not support non-namespace nested elements yet:

``` xml
<foo:config>
  <Entry key="A" value="B" />
  <Nested>
     <Entry key="A" value="B" />
  </Nested>
</foo:config>
```

When exporting it will explode with

``` plain
 Error: no namespace uri given for prefix <undefined>
      at ElementSerializer.logNamespaceUsed (.../bpmn-moddle/node_modules/moddle-xml/lib/writer.js:460:11)
      at ElementSerializer.nsTagName (.../bpmn-moddle/node_modules/moddle-xml/lib/writer.js:212:26)
      at ElementSerializer.build (.../bpmn-moddle/node_modules/moddle-xml/lib/writer.js:194:20)
      at .../bpmn-moddle/node_modules/moddle-xml/lib/writer.js:272:47
```

**Comment by nikku** [comment]
Fixed by bpmn-io/bpmn-moddle#38.

**Comment by nikku** [comment]
Fixed with latest alpha release.
